rust/scx_utils/src/compat.rs

```diff
@@ -218,7 +219,7 @@ macro_rules! scx_ops_open {
 macro_rules! scx_ops_load {
     ($skel: expr, $ops: ident, $uei: ident) => { 'block: {
         scx_utils::paste! {
-            scx_utils::uei_set_size!($skel, $ops, $uei);
+            // scx_utils::uei_set_size!($skel, $ops, $uei);
```

Comment:
For existing back ports I just skipped this macro and loaded the skels manually, but if this works.

Reply:
ended up doing the same